styx-standalone-service/src/main/resources/styx-standalone.conf

```diff
@@ -2,12 +2,12 @@ styx.mode = "production"
 
 # ttls for stale states in ISO-8601 duration format
 styx.stale-state-ttls = {
-  new            = "PT1M"
-  creating       = "PT1M"
+  new            = "PT30M"
+  creating       = "PT30M"
   submitted      = "PT10M"
   running        = "PT24H"
-  terminated     = "PT1M"
-  failed         = "PT1M"
+  terminated     = "PT30M"
+  failed         = "PT30M"
   awaiting_retry = "PT8H"
```

Comment:
Just discovered this is not really used. But let's kill it in a separated PR.

Reply:
I meant awaiting_retry. Sorry for confusion.

Reply:
Aha